wp-graphql.php

```diff
@@ -102,8 +101,7 @@ function graphql_can_load_plugin(): bool {
 		return true;
 	}
 
-	// @phpstan-ignore-next-line: this is ignored as the constant could be defined in wp-config, prior to being defined above
-	if ( ( ! defined( 'WPGRAPHQL_AUTOLOAD' ) || true === WPGRAPHQL_AUTOLOAD ) && file_exists( plugin_dir_path( __FILE__ ) . 'vendor/autoload.php' ) ) {
+	if ( file_exists( plugin_dir_path( __FILE__ ) . 'vendor/autoload.php' ) ) {
```

Comment:
The previous conditional already returns early of `WPGRAPHQL_AUTOLOAD` is not defined. And if someone's doing it wrong and defining a bad value, we ignore it.